### PR TITLE
Adding Blog Post: Top of the Open Source Charts in 2017

### PR DESCRIPTION
**Pull request by kbarnard10**
cc // @ryanj

### CODE REVIEW

File: blog/_posts/2018-04-25-open-source-charts-2017.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Read on to find out how the Kubernetes Community has addressed these scaling challenges to reach the top of the charts in GitHub’s 2017 Octoverse report.
+
+## Most-Discussed on GitHub
+
+The top two most-discussed repos of 2017 are both based on Kubernetes:
+
+![Most Discussed](/images/blog-logging/2018-04-24-open-source-charts-2017/most-discussed.png)
+
+Of all the open source repositories on GitHub, none received more issue comments than [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes/). [OpenShift](http://openshift.com/), a [CNCF certified distribution of Kubernetes](https://blog.openshift.com/kubernetes-and-openshift-community-standards-and-certifications/
+https://www.cncf.io/announcement/2017/11/13/cloud-native-computing-foundation-launches-certified-kubernetes-program-32-conformant-distributions-platforms/), took second place.
```
Comment (tengqm) [inline blog/_posts/2018-04-25-open-source-charts-2017.md]: The second link may not work

Comment (markthink) [reply blog/_posts/2018-04-25-open-source-charts-2017.md]: @tengqm  I understand that this is for version management.

**Comment by kbarnard10** [comment]
@zacharysarah Same thing with this one, needs approval from you or someone else who is an Owner.

**Comment by zacharysarah** [comment]
@kbarnard10 
/approve
/hold cancel

I wonder why @natekarcthner’s approval isn’t working. I’ll dig into it.

**Pull request merged**